ui/ui-components/components/record/AddSampleRecordForm.tsx

```diff
@@ -55,14 +55,12 @@ const AddSampleRecordForm: React.FC<Props> = ({
   const onSubmit: Parameters<typeof handleSubmit>[0] = useCallback(
     async (data) => {
       setSubmitting(true);
-      const response = await client.request<Actions.RecordCreate>(
-        "post",
-        `/record`,
-        {
+      const response = await client
+        .request<Actions.RecordCreate>("post", `/record`, {
           modelId,
           properties: { [data.uniqueProperty]: data.value },
-        }
-      );
+        })
+        .catch(() => null);
```

Comment:
https://www.pivotaltracker.com/story/show/181115484

Reply:
I agree with the latter. I'd like to see if we can figure out the issue with not caching all the errors in `client.request` first.